crates/oxc_cli/src/command.rs

```diff
@@ -235,6 +235,10 @@ pub struct WarningOptions {
     #[bpaf(switch, hide_usage)]
     pub quiet: bool,
 
+    /// Ensure warnings produce a non-zero exit code
+    #[bpaf(switch, hide_usage)]
+    pub deny_warnings: bool,
```

Comment:
Do we want `-D` as well? Might need a hand as I'm not super familiar w/ `bpaf`

Reply:
`-D` is not required, that's a `rustc` globaly flag thingy. We only need a plain cli flag for this.

Reply:
> Might need a hand as I'm not super familiar w/ bpaf

To add `-D` you can write something like  `#[bpaf(long, short('D'), switch, hide_usage)]`. Any named item (flags and arguments) takes a combination of `short` and `long`, with name being long by default.